src/applications/edu-benefits/0994/pages/highTechIndustry.js

```diff
@@ -0,0 +1,93 @@
+import fullSchema from 'vets-json-schema/dist/22-0994-schema.json';
+import _ from 'lodash';
+import { technologyEmploymentRequired } from '../utils';
+
+const {
+  currentEmployment,
+  currentHighTechnologyEmployment,
+  currentSalary,
+  highTechnologyEmploymentType,
+} = fullSchema.properties;
+
+export const uiSchema = {
+  'ui:description':
+    'To give us an idea of your experience in the high-tech market, please tell us about your work and training.',
+  currentEmployment: {
+    'ui:title': 'Are you working in a high-tech industry now?',
+    'ui:widget': 'yesNo',
+  },
+  currentHighTechnologyEmployment: {
+    'ui:title':
+      'Have you worked in a high-tech industry in the past couple years?',
+    'ui:widget': 'yesNo',
+    'ui:options': {
+      expandUnder: 'currentEmployment',
+      expandUnderCondition: false,
+    },
+    'ui:required': technologyEmploymentRequired,
```

Comment:
should this be split out into `utils.jsx`?

Reply:
is this going to be used somewhere else?

Reply:
It doesn't have to be. I can move it into here.